SNAPSHOTS.md

```diff
@@ -0,0 +1,112 @@
+# IBM Spectrum Scale CSI driver volume snapshots
```

Comment:
Maybe this should be in the docs

Reply:
Yes this should eventually go to KC. Consider this page as development input for ID team.

Reply:
Can we include the sample examples yaml files?